pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: google_fonts
 description: A package to include fonts from fonts.google.com in your flutter app.
-version: 0.2.0
+version: 0.2.1
```

Comment:
Please also update the `CHANGELOG.md` file to mention that version 0.2.1 updates all font URLs to use `https`.